deployment/modules/aws/storage/main.tf

```diff
@@ -0,0 +1,51 @@
+terraform {
+  backend "s3" {}
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+      version = "5.76.0"
+    }
+  }
+}
+
+data "aws_caller_identity" "current" {}
+
+locals {
+  name = "${var.prefix_name}-${var.base_name}"
+}
+
+# Configure the AWS Provider
+provider "aws" {
+  region = var.region
+}
+
+# Resources
+
+## S3 Bucket
+resource "aws_s3_bucket" "log_bucket" {
+  bucket = "${local.name}-bucket"
+  force_destroy = var.ephemeral
+}
+
+## Aurora MySQL RDS database
+resource "aws_rds_cluster" "log_rds" {
+  apply_immediately       = true
+  cluster_identifier      = "${local.name}-cluster"
+  engine                  = "aurora-mysql"
+  # TODO(phboneff): make sure that we want to pin this
+  engine_version          = "8.0.mysql_aurora.3.05.2"
+  database_name           = "tessera"
+  master_username         = "root"
+  master_password         = "password"
```

Comment:
Worth a TODO here to use terraform for generating random strings?

Reply:
My plan is to remove this altogether, and use IAM, see #330.
I've added a TODO here to make sure that doesn't slip away.